### ISSUE TITLE
pyvisa does not list GPIB devices on Vince

### ISSUE THREAD

**Issue by DanielSank** [martinisgroup/servers#259]
The gpib bus server on Vince cannot find any devices, and it seems to be a problem with pyvisa.

Here is a screen shot of the GUI I see if I do
```
$ cd /usr/local/bin
$ sudo ./visaconf`
```

![screenshot from 2015-11-09 11 10 18](https://cloud.githubusercontent.com/assets/4584456/11043619/7fb32756-86d2-11e5-9c23-a77115f436c8.png)

But here's what happens if I ask pyvisa to list resources

```python
In [8]: rm = visa.ResourceManager()

In [9]: rm
Out[9]: <ResourceManager(<VisaLibrary('/usr/local/vxipnp/linux/lib64/libvisa.so')>)>

In [10]: rm.list
rm.list_resources       rm.list_resources_info  

In [10]: rm.list_resources()
Out[10]: 
(u'ASRL1::INSTR',
 u'ASRL2::INSTR',
 u'ASRL3::INSTR',
 u'ASRL4::INSTR',
 u'ASRL5::INSTR',
 u'ASRL6::INSTR',
 u'ASRL7::INSTR',
 u'ASRL8::INSTR',
 u'ASRL9::INSTR',
 u'ASRL10::INSTR')
```

There are several weird things here. First, there are ten listed serial interfaces while the GUI only lists seven. Second, there are no GPIB devices while the GUI lists two.

**Comment by DanielSank** [comment]
Note that the gpib bus server doesn't even recognize devices whose names start `ASRL`:

```python
for addr in additions:
        try:
            if addr.startswith('GPIB'):
                instName = addr
            elif addr.startswith('TCPIP'):
                instName = addr
            elif addr.startswith('USB'):
                instName = addr + '::INSTR'
            else:
                continue
```

**Comment by DanielSank** [comment]
Looks like a permissions issue. Note that I'm doing this all as `daniel` and I did not sudo my ipython session. Seems like this could be a problem if the gpib bus isn't running with super user. @ejeffrey?

```python
In [11]: device = rm.open_resource('ASRL1::INSTR')
---------------------------------------------------------------------------
VisaIOError                               Traceback (most recent call last)
<ipython-input-11-94641fd6f210> in <module>()
----> 1 device = rm.open_resource('ASRL1::INSTR')

/home/daniel/.virtualenvs/temp/lib/python2.7/site-packages/pyvisa/highlevel.pyc in open_resource(self, resource_name, access_mode, open_timeout, resource_pyclass, **kwargs)
   1642                 raise ValueError('%r is not a valid attribute for type %s' % (key, res.__class__.__name__))
   1643 
-> 1644         res.open(access_mode, open_timeout)
   1645 
   1646         for key, value in kwargs.items():

/home/daniel/.virtualenvs/temp/lib/python2.7/site-packages/pyvisa/resources/resource.pyc in open(self, access_mode, open_timeout)
    201         logger.debug('%s - opening ...', self._resource_name, extra=self._logging_extra)
    202         with self._resource_manager.ignore_warning(constants.VI_SUCCESS_DEV_NPRESENT):
--> 203             self.session, status = self._resource_manager.open_bare_resource(self._resource_name, access_mode, open_timeout)
    204 
    205             if status == constants.VI_SUCCESS_DEV_NPRESENT:

/home/daniel/.virtualenvs/temp/lib/python2.7/site-packages/pyvisa/highlevel.pyc in open_bare_resource(self, resource_name, access_mode, open_timeout)
   1599         :return: Unique logical identifier reference to a session.
   1600         """
-> 1601         return self.visalib.open(self.session, resource_name, access_mode, open_timeout)
   1602 
   1603     def open_resource(self, resource_name,

/home/daniel/.virtualenvs/temp/lib/python2.7/site-packages/pyvisa/ctwrapper/functions.pyc in open(library, session, resource_name, access_mode, open_timeout)
   1209     # [ViSession, ViRsrc, ViAccessMode, ViUInt32, ViPSession]
   1210     # ViRsrc converts from (str, unicode, bytes) to bytes
-> 1211     ret = library.viOpen(session, resource_name, access_mode, open_timeout, byref(out_session))
   1212     return out_session.value, ret
   1213 

/home/daniel/.virtualenvs/temp/lib/python2.7/site-packages/pyvisa/ctwrapper/highlevel.pyc in _return_handler(self, ret_value, func, arguments)
    186 
    187         if ret_value < 0:
--> 188             raise errors.VisaIOError(ret_value)
    189 
    190         if ret_value in self.issue_warning_on:

VisaIOError: VI_ERROR_NPERMISSION (-1073807192): Access to the resource or remote machine is denied. This is due to lack of sufficient privileges for the current user or machine
```

**Comment by DanielSank** [comment]
Fixed by installing the right version of 488.2. This is documented in the relevant Drive directory and will be added to ansible.
